documentation/interactions/uirepresentation.md

```diff
@@ -185,3 +185,29 @@ dependencies {
     add("kspJs", project(":ksp:mutable-ui-processor"))
 }
 ```
+
+## Observing MotionProperty in children UI
+
+Sometimes children UI can depend on the transition animation values. Appyx provides an API to observe `MotionProperty` inside children UI.
+
+In this example, children composable retrieve and display `DpOffset` value from `Position` motion property as well as `Float` value from `RotationY` using this API:  
+
+```kotlin
+// returns dpOffset value if transition has Position MotionProperty and null otherwise 
+val dpOffset : DpOffset? = getMotionPropertyRenderValue<DpOffset, Position>()
```

Comment:
FYI depending on which gets merged first, #539 will mean it's not directly `DpOffset` that will be returned.

Reply:
I merged #539 and realised it's not just the text, but now it will also return `null` for you I guess since it's a different type.

Reply:
fixed